### PR TITLE
Restore from discovery holds

### PR DESCRIPTION
**Pull request by Batre-MSFT**
Added information and steps to restore folders from discovery holds

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeOnline/collaboration-exo/public-folders/restore-deleted-public-folder.md [hunk 1]
```diff
@@ -22,10 +22,12 @@ In some cases, you can restore a public folder that has been deleted.
 
 Public folders that have been deleted by users or administrators are stored in the public folder dumpster located in `\NON_IPM_SUBTREE\DUMPSTER_ROOT`. Deleted folders are preserved here until the time retention period is over.
 
-Any folders preserved in the public folder dumpster can be restored using EXO PowerShell. Restoring the public folder will restore all subfolders and items present in the folder.
+For scenarios where public folder contents are put on hold using retention policy, the folders reomved from `\NON_IPM_SUBTREE\DUMPSTER_ROOT` are preserved under `\NON_IPM_SUBTREE\DiscoveryHolds` till the time retention hold is over.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/collaboration-exo/public-folders/restore-deleted-public-folder.md:25]:
```suggestion
For scenarios where public folder contents are put on hold using retention policy, the folders removed from `\NON_IPM_SUBTREE\DUMPSTER_ROOT` are preserved under `\NON_IPM_SUBTREE\DiscoveryHolds` until the retention hold period is over.
```

File: Exchange/ExchangeOnline/collaboration-exo/public-folders/restore-deleted-public-folder.md [hunk 2]
```diff
@@ -22,10 +22,12 @@ In some cases, you can restore a public folder that has been deleted.
...
+
+Any folders preserved in the public folder dumpster or under DiscoverHolds folder can be restored using EXO PowerShell. Restoring the public folder will restore all subfolders and items present in the folder.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/collaboration-exo/public-folders/restore-deleted-public-folder.md:27]:
```suggestion
Any folders preserved in the public folder dumpster or under the DiscoverHolds folder can be restored using EXO PowerShell. Restoring the public folder will restore all subfolders and items present in the folder.
```

File: Exchange/ExchangeOnline/collaboration-exo/public-folders/restore-deleted-public-folder.md [hunk 3]
```diff
@@ -61,6 +69,12 @@ The user restoring the public folder must have the `Public Folders` role assigne
     Set-PublicFolder -Identity \NON_IPM_SUBTREE\DUMPSTER_ROOT\DUMPSTER_EXTEND\RESERVED_1\RESERVED_1\9f32c468-4bc2-42aa-b979-16a057394b2f\PF1 -Path \
     ```
 
+In this another example, to restore a public folder named `Sales` to the root of the public folder tree:
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/collaboration-exo/public-folders/restore-deleted-public-folder.md:72]:
```suggestion
    The following alternate example restores a public folder named `Sales` to the root of the public folder tree:
```

File: Exchange/ExchangeOnline/collaboration-exo/public-folders/restore-deleted-public-folder.md [hunk 4]
```diff
@@ -61,6 +69,12 @@ The user restoring the public folder must have the `Public Folders` role assigne
...
+
+    ```PowerShell
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/collaboration-exo/public-folders/restore-deleted-public-folder.md:74]:
```suggestion
    ```PowerShell
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review. @Batre-MSFT, please commit my suggested changes if you agree with them. any thanks.

**Comment by JohanFreelancer9** [comment]
@Batre-MSFT - Could you please commit my first suggested change as well (Any folders preserved...)? Thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@Kateyanne PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Review by lweathermon — APPROVED** [review]

**Review by lweathermon — APPROVED** [review]

**Review by lweathermon — APPROVED** [review]

**Pull request merged**